lib/forever/cli.js

```diff
@@ -196,7 +198,29 @@ function checkColumn(name) {
 // `forever.startDaemon`
 //
 var getOptions = cli.getOptions = function (file) {
-  var options = {};
+  var options = {},
+      absFile = path.isAbsolute(file) ? file : path.resolve(process.cwd(), file),
+      configKeys = [
+        'pidFile', 'logFile', 'errFile', 'watch', 'minUptime', 'append',
+        'silent', 'outFile', 'max', 'command', 'path', 'spinSleepTime',
+        'sourceDir', 'workingDir', 'uid', 'watchDirectory', 'watchIgnore',
+        'killTree', 'killSignal', 'id'
+      ],
+      jsonConfigKeys = configKeys.concat('script'),
```

Comment:
`jsonConfigKeys` is intended to be a clone of *clone* of `configKeys`, with an extra key, hence the use of `concat`. I'll look at refactoring this, regardless.